summary_workflows/identification/specification/I_benchmark_specification.md

```diff
@@ -16,14 +16,22 @@ Comparison of PAS predicted from simulated RNA-Seq data with dataset used for si
 1. Poly(A) sites identified based on simulated RNAseq data using the benchmarked tool
 2. Ground truth: Poly(A) sites of transcripts used to simulate RNAseq reads
 
+Calculating proportion of identified PAS assigned to different genomic features:
+
+1. Poly(A) sites identified based on RNAseq data using the benchmarked tool
+2. GTF file with genome annotation
+
 ### Metrics:
 
 Based on the input data the following metrics are computed:
 
 1. Sensitivity (TPR, True Positive Rate) = TP/(TP+FN)
```

Comment:
I just want to be more verbatim here to not risk misunderstandings:

- TP: number of PAS identified by participant that overlap PAS from ground truth within distance threshold.
- FP: number of PAS identified by participant that do not overlap ground truth PAS within distance threshold.
- FN: number of PAS not identified by participant but occurring in ground truth within distance threshold.
- TN: not measurable.
- distance threshold: padding of ground truth PAS in number of nucleotides in genomic positions. Example: PAS is in range [500,520] and distance threshold 10nt, so the checks for overlaps are within [490,530].
Correct @daneckaw ?

Reply:
If we take the ground truth at face value, TN are also measurable. For sure this will be the case for simulation data.

Reply:
@mzavolan, @ninsch3000  and I discussed today about TNs. The conclusion was that if the gene is not expressed in the given data, the polyA site is not identified and is thus a negative. Then a true negative would be if the prediction also does not give a polyA site on that location. 
This pattern would most likely only hold for simulation data, where from a selection of polyA sites reads are generated.

Reply:
I was thinking about it and for most cases TN would not be possible to measure, so in this version I removed the FPR metric that uses TN. Somehow I forgot to update the optional ROC curve and change it to Precision-Recall curve that does not need TN.

Reply:
I agree to leave out the TN and related measures, it's not that informative.